test/testenv/appframework_utils.go

```diff
@@ -132,3 +132,75 @@ func GetAppFileList(appList []string, version int) []string {
 	}
 	return appFileList
 }
+
+// GetBundleHash Get the status of bundle push on each indexer
+func GetBundleHash(deployment *Deployment, podName string) (string, error) {
```

Comment:
These methods only apply for CM. CSPL-1277 should also include Deployer Bundle. Please add Deployer bundle checks. Deployer bundle can be checked by `splunk list shcluster-bundle -member_uri <URI>:<management_port> -auth <username>:<password>
`. More details in docs [https://docs.splunk.com/Documentation/Splunk/8.2.2/DistSearch/PropagateSHCconfigurationchanges]